packages/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -411,6 +412,14 @@ export const GridRootStyles = styled('div', {
     },
     '@media (hover: none)': {
       [`& .${c.columnHeader}`]: columnHeaderStyles,
+      [`& .${c.columnHeader}:focus,
+        & .${c.columnHeader}:focus-within,
+        & .${c.columnHeader}:has(+ .${c.columnHeader}:focus),
+        & .${c.columnHeader}:has(+ .${c.columnHeader}:focus-within)`]: {
+        [`.${c['columnSeparator--resizable']}`]: {
+          color: (t.vars || t).palette.primary.main,
```

Comment:
What is the motivation behind having separators in an "active" state when the column header cell is selected?

Reply:
I think the colors clash a little when the column group has a blue outline and the separators are grey. This situation never occurs on desktop because the separators only grow into thicker resize handles on hover (which also turns the separator blue).

Without:
<img width="199" alt="Screenshot 2024-09-04 at 12 28 31" src="https://github.com/user-attachments/assets/bb2556ee-f103-47ff-a203-1c42a0249ebb">

With:
<img width="196" alt="Screenshot 2024-09-04 at 12 28 38" src="https://github.com/user-attachments/assets/ce98a135-8568-4712-a1cf-88682499bcfe">

Reply:
Right, this makes sense then 👍🏻